code/modules/species/species.dm

```diff
@@ -35,6 +35,12 @@ var/global/const/DEFAULT_SPECIES_HEALTH = 200
 	var/flesh_color = "#ffc896"             // Pink.
 	var/blood_oxy = 1
 
+	// Darksight handling
+	var/base_low_light_vision = 0
```

Comment:
It's never noted what the bounds for these variables are. Since we multiply by `1 - base_low_light_vision` I assume it's from 0.0 to 1.0, but nothing caps it. It might be worth checking that it's within the allowed range in `species/validate()`, since creating a new species is one of the easiest modular projects for anyone working on a Nebula downstream to try (in my opinion), and it'd be better if they got all the information in comments and CI rather than from having to ask someone who already knows.